src/geo/gmaps/gmaps-default-map-style.js

```diff
@@ -1,69 +0,0 @@
-var DEFAULT_MAP_STYLE = [{
```

Comment:
All of the Google Maps basemaps available in BUILDER have some styles, so there was no need to provide a default map style.